doc/CMakeLists.txt

```diff
@@ -17,6 +17,23 @@
 # You should have received a copy of the GNU General Public License
 # along with NEST.  If not, see <http://www.gnu.org/licenses/>.
 
+if ( NOT CMAKE_CROSSCOMPILING )
+  add_custom_target( generate_help ALL )
+  add_custom_command( TARGET generate_help POST_BUILD
+    COMMAND python -B generate_help.py "${PROJECT_SOURCE_DIR}" "${PROJECT_BINARY_DIR}"
+    WORKING_DIRECTORY "${PROJECT_SOURCE_DIR}/extras/help_generator"
+    )
+  install( DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/help"
+    DESTINATION "${CMAKE_INSTALL_DOCDIR}"
+    )
+  install( CODE
+    "execute_process(
+      COMMAND python -B generate_helpindex.py \"${CMAKE_INSTALL_DOCDIR}\"
+      WORKING_DIRECTORY \"${PROJECT_SOURCE_DIR}/extras/help_generator\"
+    )"
```

Comment:
I am a bit confused why you need to repeat the same code that is given in `add_custom_command()` here again; my confusion is probably just due to my very limited experience with CMake.